dist/dt.html

```diff
@@ -3,45 +3,51 @@
 <head>
     <meta charset="utf-8">
     <title>DigiTru.st Identity</title>
-    <script src='./digitrust.min.js'></script>
+    <!-- DT Frame Code -->
+    <script src='./digitrust_iframe.min.js'></script>
     <script>
-    // Place initializer inside a closure
+        // Place initializer inside a closure
 
         (function () {
 
-            // ATTACH GLOBAL ERROR HANDLER
-            try {
-                var errGlobal = function(msg, url, line, col, error) {
-                    var errMsg = "Digitrust frame error: " + msg;
-                    try {
-                        if (console && console.error) {
-                            console.error(errMsg);
-                        }
-                        else {
-                            console.log(errMsg);
-                        }
-
-                        var eobj = {
-                            version: 1,
-                            type: 'DigiTrust.iframe.error',
-                            value: {}
-                        }
-                        window.parent.postMessage(eobj, document.referrer);
+            // Global error handler
+            var errGlobal = function (msg, url, line, col, error) {
+                var errMsg = "Digitrust frame error: " + msg;
+                try {
+                    if (console && console.error) {
+                        console.error(errMsg);
                     }
-                    catch (myex)
-                    {
-                        console.log(myex);
+                    else {
+                        console.log(errMsg);

```

Comment:
change to Logger.error() as well?

Reply:
Ditto here. Will add to for Logger access.

Reply:
agree